bin/hdfs-mesos

```diff
@@ -1,5 +1,39 @@
 #!/bin/bash
 
+FRAMEWORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
+
+export MESOS_HDFS_DATA_DIR=${MESOS_HDFS_DATA_DIR:-"/tmp/hdfs/data"}
```

Comment:
Ah, true. You can choose to overwrite the env variables and just change the mesos-site.xml directly. Thanks for updating the readme.